components/widgets/badges/badges_spec.js

```diff
@@ -15,7 +15,8 @@
 /* eslint-disable max-nested-callbacks */
 
 describe('Widgets - Badges', () => {
-    before(() => {
+    beforeEach(() => {
```

Comment:
My worry here is the big shift on how we think about `before` hook and this change seemed to suggests to totally forget on using it which I'm 3/5 not to go in that direction.